src/ecdsa/robust_ecdsa/presign.rs

```diff
@@ -173,19 +165,31 @@ async fn do_presign(
         .into_vec_or_none()
         .ok_or(ProtocolError::InvalidInterpolationArguments)?;
 
-    // polynomial interpolation of w
-    let w = Polynomial::eval_interpolation(&identifiers, &signingshares, None)?;
-
     // exponent interpolation of big R
-    let identifiers: Vec<Scalar> = verifyingshares_map
-        .participants()
-        .iter()
-        .map(|p| p.scalar::<C>())
-        .collect();
     let verifying_shares = verifyingshares_map
         .into_vec_or_none()
         .ok_or(ProtocolError::InvalidInterpolationArguments)?;
 
+    #[cfg(feature = "actively_secure_ecdsa")]
+    {
+        // check that the exponent interpolations match what has been received
+        for i in threshold + 1..identifiers.len() {
+            let p = &identifiers[i];
+            // exponent interpolation for (R0, .., Rt; i)
+            let big_r_j = PolynomialCommitment::eval_exponent_interpolation(
```

Comment:
```suggestion
            let big_r_i = PolynomialCommitment::eval_exponent_interpolation(
```
Else I dont know where is `j` coming from

Reply:
Was just following the doc's notation